### PR TITLE
Improved Files Resubmitted

### PR DESCRIPTION
**Pull request by coatexblee**
Maplist JS script and stylesheet submitted plus improved HTML files for accessibility.

### CODE REVIEW

File: resources.html [hunk 1]
```diff
@@ -225,14 +227,14 @@ <h4 class="media-heading text-uppercase reviews">Neighborhood Research</h4>
...
                                 <ul class="resource-links">
                                     <!-- <li><a class="btn btn-info btn-circle text-uppercase" href="http://austin.maps.arcgis.com/apps/webappviewer/index.html?id=d7c4532780aa431ca7f523a57426d402" id="reply" target="_blank">City Owned Property</a><br>Map of city-owned property AKA which department has stewardship over the property. Note: The City does not own property, they manage it - often referred to as stewardship. When someone asks which department “owns” a parcel, they actually mean which department has stewardship, a careful distinction that may or may not affect a court case.</li> -->
-                                    <li><a class="btn btn-info btn-circle text-uppercase" href="http://austin.maps.arcgis.com/apps/webappviewer/index.html?id=d7c4532780aa431ca7f523a57426d402" id="reply" target="_blank">City Owned Property</a><br>Map of city-owned
+                                    <li><a class="btn btn-info btn-circle text-uppercase" href="http://austin.maps.arcgis.com/apps/webappviewer/index.html?id=84614e5acde342eaa640a97c3269b335" id="reply" target="_blank">City Owned Property</a><br>Map of city-owned
```
Comment (NateATXCode) [inline resources.html:237]: any idea why this was changed?

File: resources.html [hunk 2]
```diff
@@ -18,11 +18,13 @@
...
     <!-- Custom styles for this template -->
     <link href="css/custom-theme.css" rel="stylesheet">
+    <!-- Modified the navbar links for accessbility-->
+    <link href="css/navbarcustom.css" rel="stylesheet">
     <!-- js files -->
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
     <script type="text/javascript" src="js/bootstrap.min.js"></script>
     <script type="text/javascript" src="js/custom-script.js"></script>
-    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="shortcut icon" href="favicon2.ico" type="image/x-icon">
```
Comment (NateATXCode) [inline resources.html:27]: this is fine, but why do you think there are multiple favicon files? we should be cleaner about that. too many duplicates lying around basically

File: demographics.html [hunk 3]
```diff
@@ -67,150 +65,102 @@ <h1>Demographics</h1>
...
+      							<option value="W5">W5</option>
+      							<option value="W6">W6</option>
+      							<option value="W7">W7</option>
+      							<option value="W8">W8</option> -->
       						</select>
                     </div>
                 </div>
             </div>
             <br>
             <!--Charts-->
```
Comment (NateATXCode) [inline demographics.html:112]: why did the below get removed? did you check the code locally?

File: demographics.html [hunk 4]
```diff
@@ -12,24 +12,22 @@
...
-    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
-    <style>
-        .sources {
-            font-size: 9px;
-        }
-        div.row {
-            margin: 30px 0px 30px 0px;
-        }
-    </style>
+    <!-- js files -->
```
Comment (NateATXCode) [inline demographics.html:24]: i think this is from an older repository still

**Review by NateATXCode — COMMENTED** [review]
going to close this so we can try a better pull request process.

**Comment by coatexblee** [comment]
I’m not sure what’s going on with ArcGIS links’ ids in resources.html. I haven’t even touch it. It looks like database within the server automatically reassign ids.    About the multiple favicons. I kept the original favicon in the file along with the new one with different but similar name. In case my modified favicon is not accepted, we could always go back to the original one, and create new one in the future if any one of use has improved artistic skill. I didn’t delete the codes below the raw HTML coded select tags. They were already there, unless you might had made few changes to the original  file you pull back in time  before merging my files. I notice there some differences between  the new cloned repository of yours and the original improved demographics.html I have. Yes, I worked the codes locally.

**Pull request closed without merging**